lib/utilities/reportHandler.js

```diff
@@ -323,6 +475,8 @@ function reportHandler(clientIP, req, res, log) {
                 repStatus: results.getReplicationStates,
                 config: cleanup(config),
                 capabilities: getCapabilities(),
+                ingestStats: results.getIngestionInfo.metrics,
```

Comment:
if on error, do you expect the value here to be an empty object or undefined?

Reply:
returning empty object

Reply:
I believe empty object if you see other responses.